contrib/ansible/deploy-playbook.yaml

```diff
@@ -44,23 +50,30 @@
       redeploy_apiserver_cert: True
     when: apiserver_cert_exists_reg.rc > 0 and "NotFound" in apiserver_cert_exists_reg.stderr
 
-  # TODO: replace with openssl_certificate but watch out for need for a CA
-  - name: generate apiserver certs
-    command: "{{ playbook_dir }}/../apiserver-aggregation-tls-setup.sh"
-    args:
-      # ensure these land in the top level of the project where expected
-      chdir: "{{ playbook_dir }}/../../"
-      # only runs if this file does not exist in top level of the git repo.
-      creates: "{{ playbook_dir }}/../../apiserver.pem"
-    when: redeploy_apiserver_cert | bool
+  - include_role:
+      name: cluster_operator_certgen
+    vars:
+      cert_dir: "{{ apiserver_cert_dir }}"
+      ca_name: "{{ apiserver_ca_name }}"
+      cert_name: "apiserver"
+      cn: cluster-operator-apiserver
+      alt_names: "cluster-operator-apiserver.{{ cluster_operator_namespace }}\",\"cluster-operator-apiserver.{{ cluster_operator_namespace }}.svc"
```

Comment:
same comment as above

Reply:
Not sure just following what was in the apiserver cert gen script.